Bigtable/tests/System/BigtableInstanceAdminClientTest.php

```diff
@@ -45,6 +44,9 @@ public static function setUpBeforeClass()
             return;
         }
 
+        self::setUsingEmulator(getenv('BIGTABLE_EMULATOR_HOST'));
```

Comment:
This class is generated, so this change won't stick.

Reply:
Is it? There's no generated code warning.
If it's true, we need to slightly change `bigtable-emulator-system-tests.yaml` (add `--group bigtable` parameter for run tests step).

Reply:
Hmm, maybe not in this case. Anyway, let's just make that change and avoid modifying this file.